packages/components/src/components/hds/stepper/list/step.ts

```diff
@@ -0,0 +1,156 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import { assert } from '@ember/debug';
+import { guidFor } from '@ember/object/internals';
+import { action } from '@ember/object';
+import { modifier } from 'ember-modifier';
+
+import {
+  HdsStepperStatusesValues,
+  HdsStepperTitleTagValues,
+} from '../types.ts';
+import {
+  type HdsStepperStatuses,
+  type HdsStepperTitleTags,
+  type HdsStepperListStepIds,
+  HdsStepperStatusToSrOnlyText,
+} from '../types.ts';
+
+export const DEFAULT_STATUS = HdsStepperStatusesValues.Incomplete;
+export const STATUSES: string[] = Object.values(HdsStepperStatusesValues);
+export const MAPPING_STATUS_TO_SR_ONLY_TEXT = HdsStepperStatusToSrOnlyText;
+
+export interface HdsStepperListStepSignature {
+  Args: {
+    status: HdsStepperStatusesValues;
+    titleTag?: HdsStepperTitleTags;
+    stepIds?: HdsStepperListStepIds;
+    didInsertNode?: (element: HTMLElement) => void;
+    willDestroyNode?: (element: HTMLElement) => void;
+  };
+  Blocks: {
+    title: [];
+    description?: [];
+    content?: [];
+  };
+  Element: HTMLElement;
+}
+
+export default class HdsStepperListStep extends Component<HdsStepperListStepSignature> {
+  /**
+   * Generate a unique ID for the Step
+   * @return {string}
+   * @param _stepId
+   */
```

Comment:
We're not using this style anymore. You could replace with a simple one-line comment like:

```
// Generate a unique ID for the Step
```